themes/classic/templates/_partials/breadcrumb.tpl

```diff
@@ -23,19 +23,19 @@
  * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
  *}
 <nav data-depth="{$breadcrumb.count}" class="breadcrumb hidden-sm-down">
-  <ol itemscope itemtype="https://schema.org/BreadcrumbList">
+  <ol>
     {block name='breadcrumb'}
       {foreach from=$breadcrumb.links item=path name=breadcrumb}
         {block name='breadcrumb_item'}
           {if not $smarty.foreach.breadcrumb.last}
-            <li itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
-              <a itemprop="item" href="{$path.url}"><span itemprop="name">{$path.title}</span></a>
-              <meta itemprop="position" content="{$smarty.foreach.breadcrumb.iteration}">
+            <li>
```

Comment:
You can put the li and the iteration before the condition... more simple :-)

Reply:
oh it's from the classic template in develop branch I didn't change anything to don't overload the pr.. I only removed the itemprop and itemscope in the tpl(s)

Reply:
@fdonnet What do you think of this? The iteration does not even have to be there.

```
<nav data-depth="{$breadcrumb.count}" class="breadcrumb hidden-sm-down">
  <ol>
    {block name='breadcrumb'}
      {foreach from=$breadcrumb.links item=path name=breadcrumb}
        {block name='breadcrumb_item'}
          <li>
            {if not $smarty.foreach.breadcrumb.last}
                <a href="{$path.url}"><span>{$path.title}</span></a>
            {elseif isset($path.title)}
                <span>{$path.title}</span>
            {/if}
          </li>
        {/block}
      {/foreach}
    {/block}
  </ol>
</nav>
```

Reply:
yes, for me it's cleaner... but like your comment below... if you said I can remove the meta in html flow I will do it everywhere and particulary in the product.tpl (at the top of the tpl and in the flow)... and keep all the declarations in the json-md file. (and i will remove the 3 item prop I added to try to pass the test too)... ;)

maybe the file name of the new .tpl (microdata-jsonld.tpl) needs to be reviewed to show we put meta tag in it too :)

Reply:
discussion continues above / other thread ;)